src/javascript/upload/view.js

```diff
@@ -143,22 +143,65 @@ Upload.prototype._processResponse = function (err, inkBlob) {
         return;
     }
 
-    var contents = [];
     var self = this;
     if (inkBlob) {
         if (!inkBlob.length) {
             inkBlob = [inkBlob];
         }
 
-        $.each(inkBlob, function (i, blob) {
+        $.each(inkBlob, function (index, blob) {
+            self._postProcess(blob);
+        });
+    }
+};
+
+/**
+ * @param {Object} blob
+ * @private
+ */
+Upload.prototype._postProcess = function (blob) {
+    var self = this;
+
+    var converters = {
+        image: function (blob, done) {
             picker.convert(blob, self.opts.convert, self.opts.store, function (convertedBlob) {
-                var content = self._packageInput(convertedBlob);
-                contents.push(content);
-                self.writeToDestination(content);
+                var url = self._cacheUrl + convertedBlob.key;
+                var attachment = {
+                    type: 'photo',
+                    url: url,
+                    link: url,
+                    provider_name: 'Livefyre'
+                    //TODO (joao) images dimensions?!!!!
+                };
+
+                var content = new Content({body: ''});
+                content.attachments.push(attachment);
+                done(content);
             });
-        });
+        },
+
+        video: function (blob, done) {
+            var url = self._cacheUrl + blob.key;
```

Comment:
This is only used once, I'd say just plug it in instead of having an extra variable.